lib/ClangImporter/ClangImporter.cpp

```diff
@@ -7597,9 +7596,53 @@ bool importer::hasEscapableAttr(const clang::RecordDecl *decl) {
   return hasSwiftAttribute(decl, "Escapable");
 }
 
+static const llvm::StringMap<std::vector<int>> STLConditionalEscapableParams{
+    {"vector", {0}},
+    {"array", {0}},
+    {"inplace_vector", {0}},
+    {"deque", {0}},
+    {"forward_list", {0}},
+    {"list", {0}},
+    {"set", {0}},
+    {"flat_set", {0}},
+    {"unordered_set", {0}},
+    {"multiset", {0}},
+    {"flat_multiset", {0}},
+    {"unordered_multiset", {0}},
+    {"stack", {0}},
+    {"queue", {0}},
+    {"priority_queue", {0}},
+    {"tuple", {0}},
+    {"variant", {0}},
+    {"optional", {0}},
+    {"pair", {0, 1}},
+    {"expected", {0, 1}},
+    {"map", {0, 1}},
+    {"flat_map", {0, 1}},
+    {"unordered_map", {0, 1}},
+    {"multimap", {0, 1}},
+    {"flat_multimap", {0, 1}},
+    {"unordered_multimap", {0, 1}},
+    {"span", {0}},   // TODO: remove when span is escapable by default
+    {"mdspan", {0}}, // TODO: remove when mdspan is escapable by default
```

Comment:
Hmm, isn't span currently imported as escapable in Swift?

Reply:
Yup. The comment supposed to be: remove when span is non-escapable by default. I will fix this up.